rust/alloc/raw_vec.rs

```diff
@@ -218,6 +235,29 @@ impl<T, A: Allocator> RawVec<T, A> {
         }
     }
 
+    fn try_allocate_in(capacity: usize, init: AllocInit, alloc: A) -> Result<Self, TryReserveError> {
```

Comment:
You might want to replace body `allocate_in` with `handle_reserve(Self::try_allocate_in(capacity, init, alloc))`.

Reply:
I think we can do that when upstreaming the change, to avoid increasing the diff here (and keep it as additions-only wherever possible).